src/ToolBox/superpmi/mcs/verbmerge.h

```diff
@@ -15,21 +16,23 @@ class verbMerge
     static int DoWork(const char* nameOfOutputFile, const char* pattern, bool recursive);
 
 private:
-    typedef bool (*DirectoryFilterFunction_t)(WIN32_FIND_DATAA*);
-    static bool DirectoryFilterDirectories(WIN32_FIND_DATAA* findData);
-    static bool DirectoryFilterFile(WIN32_FIND_DATAA* findData);
-    static int __cdecl WIN32_FIND_DATAA_qsort_helper(const void* p1, const void* p2);
-    static int FilterDirectory(const char*                  searchPattern,
+    typedef bool (*DirectoryFilterFunction_t)(WIN32_FIND_DATAW*);
+    static bool DirectoryFilterDirectories(WIN32_FIND_DATAW* findData);
+    static bool DirectoryFilterFile(WIN32_FIND_DATAW* findData);
+    static int __cdecl WIN32_FIND_DATAW_qsort_helper(const void* p1, const void* p2);
+    static int FilterDirectory(LPCTSTR                      searchPattern,
                                DirectoryFilterFunction_t    filter,
```

Comment:
Here, and elsewhere, you should use LPCWSTR, LPWSTR, and WCHAR, not the "T" variants. (The T variants switch between CHAR and WCHAR depending on the UNICODE define. We don't want these to shift).